ui/StatusQ/src/StatusQ/Controls/StatusPasswordStrengthIndicator.qml

```diff
@@ -74,7 +74,7 @@ StatusProgressBar {
 
        Default value: "So-so"
```

Comment:
You could just remove that line though, it's not a useful comment
```suggestion
       Default value: "Okay"
```

Reply:
Yeah, will remove